### PR TITLE
Update to work with latest version of Klaytn

### PR DESCRIPTION
**Pull request by 2dvorak**
## Proposed changes

- Update README to provide examples
- Correct paths 413ac470553397fca6a83f97b8760ebc6dedcc26
    - Copy `static-nodes.json` to correct path
    - Place `nodekey` file to correct path
- Add missing conditions d70330069fd2e8d75efd895972f30814b50d05e1
    - Running homi is only for builder
    - Setting ansible path is for remote nodes
- Add port number after address in KNIs for `static-nodes.json` 78dbb57b1a91227208a15b23b3f2e1801ad412e1
- Misc
    - Fix typo 0369d89d768782e9fc747b8e8cdbc6abe156b9ec d6deedb5f839f217d37959c44d8cfab7741f3978
    - Add missing `remote_src`
    - Removed some trailing whitespaces

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

- I have tested the changes with 4 SCNs on AWS.

## Related issues

- #1 

## Further comments

Because this repository does not have any template for pull request, I copied [one](https://github.com/klaytn/klaytn/blob/dev/.github/PULL_REQUEST_TEMPLATE.md) from the [klaytn](https://github.com/klaytn/klaytn) repository.

### CODE REVIEW

File: roles/klaytn_node/defaults/main.yml [hunk 1]
```diff
@@ -128,16 +129,16 @@ klaytn_network_BOOTNODES: ""
...
 
 # Klaytn blockchain data folder path
 klaytn_node_DATA_DIR: "/var/kcnd/data"
 
 # Log folder path
 klaytn_node_LOG_DIR: "/var/kcnd/log"
 
-# NodeKey path for Bootnode
-klaytn_node_KEY_PATH: "{{klaytn_node_DATA_DIR}}/nodekey"
+# NodeKey foler path
```
Comment (baerae-colin) [inline roles/klaytn_node/defaults/main.yml]:
```suggestion
# NodeKey folder path
```

**Review by jaydenhur — APPROVED** [review]

**Comment by jaydenhur** [comment]
@2dvorak thank you for correcting all typos and adding installation/configure part without using yum pkg manager. can you plz tell me which linux and pkg manager had you have tested the ansible? In previous, it was tested in Amazon Linux 2 and yum pkg manager only.

**Comment by 2dvorak** [comment]
@jaydenhur I tested with CentOS and explicitly set ansible_facts.pkg_mgr to another string (e.g., "noyum"). That way, I could make ansible to manually download Klaytn binaries. As I don't have permission to add subscription to other AWS marketplace images, I could not test with other images.

**Review by iv0rish — APPROVED** [review]
LGTM 👍

**Pull request merged**